synapse/push/push_tools.py

```diff
@@ -49,7 +50,33 @@ async def get_badge_count(store: DataStore, user_id: str, group_by_room: bool) -
 async def get_context_for_event(
     storage: StorageControllers, ev: EventBase, user_id: str
 ) -> Dict[str, str]:
```

Comment:
Historical problem but this function name is confusing because `EventContext` is also a thing.

And it would be nice to return a dataclass here so it's obvious what keys are available.